vagrant/README.md

```diff
@@ -12,16 +12,28 @@
 
     cd <vespa-source>/vagrant
 
-#### 2. Install Vagrant VirtualBox Guest Additions plugin
+#### 2. Choose dev environment
+
+##### a. For a dev environment with plain centos/7 and no GUI:
+
+    export VESPA_VAGRANT_VM_BOX=centos/7
+    export VESPA_DISABLE_VAGRANT_GUI=true
+
+##### b. For a dev environment with GUI and CLion:
+
+    export VESPA_VAGRANT_VM_BOX=<TODO>
+    export VESPA_VAGRANT_VM_BOX_URL=<TODO>
```

Comment:
Sub-section for building the box would contain something like this:

Create centos7-desktop box

1) Install packer
Follow guide at https://www.packer.io/intro/getting-started/install.html.

2) Clone boxcutter centos repo
git clone https://github.com/boxcutter/centos.git

3) Build box
./bin/box build centos7-desktop.json virtualbox

Example exports:
export VESPA_VAGRANT_VM_BOX="centos7-desktop"
export VESPA_VAGRANT_VM_BOX_URL="/Users/foo/git/boxcutter/centos/box/virtualbox/centos7-desktop-xx.yyyy.z.box"